package.json

```diff
@@ -19,8 +19,19 @@
         "types": "./dist/esm/lib/index.d.ts",
         "default": "./dist/esm/lib/index.js"
       }
+    },
+    "./models/*": {
```

Comment:
If we export models like this I think we should also export preconfigured and rulesets